frontend/src/metabase/dashboard/actions/cards.js

```diff
@@ -100,6 +101,11 @@ export const addDashCardToDashboard = function ({
 };
 
 export const addMarkdownDashCardToDashboard = function ({ dashId, tabId }) {
+  trackSchemaEvent("dashboard", "1-2-0", {
```

Comment:
We usually create `analytics.ts` file in the feature folder and add calls to analytics there. It makes it easier to understand what events are tracked in this feature. Example https://github.com/metabase/metabase/blob/03a80bbf1432c57438ce9173caac7f4fd3cf397b/frontend/src/metabase/setup/analytics.ts#L3-L2